src/Control/Director.php

```diff
@@ -873,6 +873,11 @@ protected static function force_redirect($destURL)
      */
     public static function forceSSL($patterns = null, $secureDomain = null)
     {
+        // if no request i.e. has been set in _config.php - prevents loops
+        if (!static::currentRequest()) {
+            return false;
```

Comment:
Can you make this a warning? If you hit this it's really an error condition that should be fixed in code.